src/components/AboutUs/Members/Members.jsx

```diff
@@ -1,23 +1,43 @@
-import React from "react";
+import React, { useEffect } from "react";
+import PropTypes from "prop-types";
+import Member from "./Member";
 import { MEMBERS_TITLE } from "../../../constants/AboutUs";
-import MEMBERS from "../../../mockup-db/aboutUsMembers";
 import "./Members.scss";
 
-const Members = () => (
-  <div className="about-members">
-    <h3 className="members-title">{MEMBERS_TITLE}</h3>
-    <div className="members-container">
-      {MEMBERS.map(({ name, description, img }) => (
-        <div className="member" key={name}>
-          <img className="member__image" src={img} alt={name} />
-          <div className="member__info">
-            <h2 className="member__name">{name}</h2>
-            <p className="member__description">{description}</p>
-          </div>
-        </div>
-      ))}
+const Members = ({ aboutUs: { members, loading, error }, getMembers }) => {
+  useEffect(() => {
+    if (members.length === 0) {
+      getMembers();
+    }
+  }, [members, getMembers]);
+
+  if (loading || error) {
+    return null;
+  }
+  return (
+    <div className="about-members">
+      <h3 className="members-title">{MEMBERS_TITLE}</h3>
+      <div className="members-container">
+        {members.map(({ id, name, description, img }) => (
+          <Member key={id} name={name} description={description} img={img} />
+        ))}
+      </div>
     </div>
-  </div>
-);
+  );
+};
```

Comment:
There is a Spinner wrapper-component, widely used in project.
It will be good if use it and avoid nested props and unneeded logic via conditions